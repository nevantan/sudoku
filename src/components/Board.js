import React from 'react';
import Cage from './Cage';

const Board = () => (
  <div>
    <p>Board Component</p>
    <Cage />
    <Cage />
    <Cage />
    <Cage />
    <Cage />
    <Cage />
    <Cage />
    <Cage />
    <Cage />
  </div>
);

export default Board;
