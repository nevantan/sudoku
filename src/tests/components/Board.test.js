import React from 'react';
import { shallow } from 'enzyme';
import Board from '../../components/Board';

test('should render Board component correctly', () => {
  const wrapper = shallow(<Board />);
  expect(wrapper).toMatchSnapshot();
});
