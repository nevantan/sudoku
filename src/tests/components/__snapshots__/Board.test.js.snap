// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Board component correctly 1`] = `
<div>
  <p>
    Board Component
  </p>
  <Cage />
  <Cage />
  <Cage />
  <Cage />
  <Cage />
  <Cage />
  <Cage />
  <Cage />
  <Cage />
</div>
`;
